lib/zendesk_apps_support/package.rb

```diff
@@ -365,5 +371,12 @@ def read_json(path, parser_opts = {})
       file = read_file(path)
       JSON.parse(read_file(path), parser_opts) unless file.nil?
     end
+
+    def text_file?(file)
```

Comment:
nit: can we push this check down into `validations/secrets.rb`, since it's not being used anywhere else?

Reply:
I wanted `text_files` to live alongside its other file filter method friends in this class (and so also defined `text_file?` in the same file), but given https://github.com/zendesk/zendesk_apps_support/pull/226/files/d28b0b1cd7ad21a48bc8026807c3edc762c5a46a#r284574646 I'll probably just remove this altogether